tests/Microsoft.Identity.Test.Unit/PublicApiTests/ConfidentialClientApplicationTests.cs

```diff
@@ -1080,6 +1080,55 @@ public void EnsurePublicApiSurfaceExistsOnInterface()
             PublicClientApplicationTests.CheckBuilderCommonMethods(byRefreshTokenBuilder);
         }
 
+        [TestMethod]
+        public async Task ConfidentialClientSuggestedExpiryAsync()
```

Comment:
I'm not sure this tests the right thing. We should have 2 tests around expiry: 

Token cache has AT1 with exp 1h, AT2 with exp 2h and AT3 with exp 3h

1. new AT comes in, with expiry 1.5h. expectated expiry ~3h 
2. new AT comes in with expirty 0.5h -> ~3h
3. new AT with 5h -> ~5h
etc.